packages/stark-ui/src/modules/table/components/table.component.ts

```diff
@@ -513,18 +537,30 @@ export class StarkTableComponent extends AbstractStarkUiComponent implements OnI
 		}
 
 		if (changes["rowsSelectable"]) {
-			if (this.rowsSelectable) {
-				if (!this.displayedColumns.includes("select")) {
-					this.displayedColumns.unshift("select");
-				}
+			if (this._managedSelection) {
+				this.logger.error(
+					`${componentName}: 'selection' cannot be used with 'multiSelect' and/or 'rowsSelectable'. Please use 'selection' only.`
+				);
 			} else {
-				const i: number = this.displayedColumns.indexOf("select");
-				this.displayedColumns.splice(i);
+				if (this.rowsSelectable) {
+					if (!this.displayedColumns.includes("select")) {
```

Comment:
could you combine these if statements

Reply:
I'm not a big fan of combining those 2 if statements. 
What would happen if `rowsSelectable` is set twice to `true` (`true` or `"true"` or 1 using `any` cast) ?
The `"select"` displayedColumn would be deleted even it's not the goal...

What do you think ?

Reply:
Ah yes, I missed that there is a third "else do nothing" clause and I kind of missed what the code was doing. 
If I may suggest a small refactor. But that's mostly preference, so I won't block the PR for this.

```TS
const i = this.displayedColumns.indexOf("select");

// Show column
if (this.rowsSelectable && i === -1) {
  this.displayedColumns.unshift("select");
}

// Hide column
if (!this.rowsSelectable && i >= 0) {
  this.displayedColumns.splice(i);
}
```

Reply:
For the rc.1, I'll do a PR to cleanup a bit the table component 😊